src/store/page/page-element.ts

```diff
@@ -157,7 +158,7 @@ export class PageElement {
 		if (json && (json as JsonObject)['_type'] === 'pattern') {
 			return PageElement.fromJsonObject(json as JsonObject, store, this);
 		} else {
-			const styleguide = store.getStyleguide();
+			const styleguide = store.getStyleguide() as Styleguide;
```

Comment:
This type cast could probably also be removed and replaced with a check if the styleguide is undefined

Reply:
As discussed, we leave it as it is, before in this case, we can ensure the styleguide always exists.